client/types/render.go

```diff
@@ -447,21 +452,30 @@ func (eis emptyIngesterStates) MarshalJSON() ([]byte, error) {
 	return json.Marshal([]ingest.IngesterState(eis))
 }
 
+type emptyMissingIngester []MissingIngester
```

Comment:
not wild about this naming although i acknowledge it's internal

Reply:
I agree, its just for the marshalling garbage

I wish so much i could just annotate the field and say "not null"